app/src/main/res/layout/content_event.xml

```diff
@@ -555,14 +555,14 @@
                     android:visibility="gone"
                     tools:visibility="visible"/>
 
-                <Button
-                    android:id="@+id/feedbackBtn"
+                <TextView
```

Comment:
Why?

Reply:
TextView is looking good instead of the button same as tickets

Reply:
But it is not a Button, also it is horrible for accesability. Secondly, it won't be selected as a button on navigration through keyboard

Reply:
But why it requires navigation through the keyboard. It is displaying an alert dialog to write feedback. Then the dialog button will submit the feedback.

Reply:
Everything should be accessible. If you're changing a button to textview, why not do it for all layouts? Let's open a merge request in AOSP to remove Button class. Let's deprecate it in favor of TextView. Why do we even need button when there is TextView?

Reply:
https://material.io/design/usability/accessibility.html#implementing-accessibility

Reply:
https://developer.android.com/guide/topics/ui/accessibility/apps.html#navigation

Reply:
You can try
`            style="@style/Widget.MaterialComponents.Button.TextButton"
`
@liveHarshit

Reply:
It needs to be a Button

Reply:
Agree with you. There are other text views also using for click listeners. Should I replace them also?

Reply:
Yes